pkg/aws/cloud.go

```diff
@@ -66,7 +76,7 @@ func NewCloud(cfg CloudConfig, metricsRegisterer prometheus.Registerer) (Cloud,
 		cfg:      cfg,
 		appMesh:  services.NewAppMesh(sess),
 		cloudMap: services.NewCloudMap(sess),
-		eks:      services.NewEKS(sess),
+		eks:      services.NewEKS(sessEKS),
```

Comment:
I know EKS does not have dual stack endpoints, but it does have FIPS. This does not seem like it should ignore the FIPS endpoints

Reply:
Can we create a separate config for EKS that can set FIPS but not dual stack?